fachhochschule-vorarlberg-author-date.csl

```diff
@@ -165,25 +165,19 @@
   </macro>
   <macro name="access">
     <choose>
-      <if type="webpage post-weblog" match="any">
-        <choose>
-          <if variable="URL">
-            <group delimiter=" " suffix=":">
-              <text term="online" text-case="capitalize-first"/>
-              <text value="im"/>
-              <text term="internet"/>
-            </group>
-            <text variable="URL"/>
-            <group prefix=" (" delimiter=" " suffix=").">
-              <text term="accessed" suffix=":"/>
-              <date variable="accessed">
-                <date-part name="day" form="numeric-leading-zeros" suffix="."/>
-                <date-part name="month" suffix="." form="numeric-leading-zeros"/>
-                <date-part name="year" form="long"/>
-              </date>
+		<if variable="URL">
+			<group prefix=" " suffix=": ">
+			  <text value="Online im Internet"/>
             </group>
-          </if>
-        </choose>
+			<text variable="URL"/>
+			<group prefix=" (" delimiter=" " suffix=").">
+			  <text value="Zugriff am:"/>
```

Comment:
you can use the variable "accessed" instead of the hardcoded text. See the locale section for reference (line ~40)